src/App/Datums.cpp

```diff
@@ -144,11 +149,14 @@ LocalCoordinateSystem::LocalCoordinateSystem()
 
 LocalCoordinateSystem::~LocalCoordinateSystem() = default;
 
-bool LocalCoordinateSystem::getCameraAlignmentDirection(Base::Vector3d& direction,
+bool LocalCoordinateSystem::getCameraAlignmentDirection(Base::Vector3d& directionZ,
+                                                        Base::Vector3d& directionX,
                                                         const char* subname) const
 {
     Q_UNUSED(subname);
-    Placement.getValue().getRotation().multVec(Base::Vector3d(0., 0., 1.), direction);
+
+    Placement.getValue().getRotation().multVec(Base::Vector3d(0., 0., 1.), directionZ);
```

Comment:
Same goes here, maybe make named var Horizontal and Vertical

Reply:
I also changed this to `Base::Vector3d::UnitZ` and `Base::Vector3d::UnitX`.